### PR TITLE
Rename args and fields after HTTPFormat

### PR DESCRIPTION
**Pull request by rakyll**

### CODE REVIEW

File: plugin/http/httptrace/httptrace.go [hunk 1]
```diff
@@ -26,8 +26,8 @@ import (
 // TODO(jbd): Add godoc examples.
 
 type transport struct {
-	base        http.RoundTripper
-	propogators []propagation.HTTPFormat
+	base      http.RoundTripper
+	formaters []propagation.HTTPFormat
```
Comment (odeke-em) [inline plugin/http/httptrace/httptrace.go]: s/formater/formatter/g

File: plugin/http/httptrace/httptrace.go [hunk 2]
```diff
@@ -86,8 +86,8 @@ func NewTransport(base http.RoundTripper, p ...propagation.HTTPFormat) http.Roun
 // The span will be automatically ended by the handler.
 //
 // Incoming propagation mechanism is determined by the given HTTP propagators.
-func NewHandler(base http.Handler, p ...propagation.HTTPFormat) http.Handler {
-	return &handler{handler: base, propogators: p}
+func NewHandler(base http.Handler, format ...propagation.HTTPFormat) http.Handler {
+	return &handler{handler: base, propogators: format}
```
Comment (nvartolomei) [inline plugin/http/httptrace/httptrace.go]: shouldn't `propagators` field of handler struct be renamed too, for consistency?

Comment (rakyll) [reply plugin/http/httptrace/httptrace.go]: Done.

**Comment by rakyll** [comment]
Renamed formatters to formats instead.

File: plugin/http/httptrace/httptrace.go [hunk 3]
```diff
@@ -104,7 +104,7 @@ func (h *handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	)
 
 	ctx := r.Context()
-	for _, p := range h.propogators {
+	for _, p := range h.formats {
```
Comment (nvartolomei) [inline plugin/http/httptrace/httptrace.go]: one nit, rename `p` to `f` :)

Comment (rakyll) [reply plugin/http/httptrace/httptrace.go]: Done, thanks much.

**Pull request merged**